CMakeLists.txt

```diff
@@ -111,14 +110,28 @@ endif()
 add_library(mimalloc SHARED ${mi_sources})
 set_target_properties(mimalloc PROPERTIES VERSION ${mi_version} NO_SONAME "YES" OUTPUT_NAME ${mi_basename} )
 target_compile_definitions(mimalloc PRIVATE ${mi_defines} MI_SHARED_LIB MI_SHARED_LIB_EXPORT)
+target_compile_definitions(mimalloc PRIVATE $<$<BOOL:${OVERRIDE}>:MI_MALLOC_OVERRIDE>)
```

Comment:
It might be easier to read for non-CMake experts (such as myself) if there was an `if(OVERRIDE MATCHES "ON")...endif` guard around a `list(APPEND mi_defines MI_MALLOC_OVERRIDE` here...

Reply:
Hey! @dscho Thanks for your kindly review. I have updated the PR. :-)